### ISSUE TITLE
WCAG 2.4.6: Headings and Labels (Dashboard)

### ISSUE THREAD

**Issue by brianteeman** [joomla-projects/j4adminui#68]
### Steps to reproduce the issue
Either examine the source or open the skipto menu

### Expected result
Headings are sequential with no missed steps
Skipping heading ranks can be confusing and should be avoided where possible: Make sure that a <h2> is not followed directly by an <h4>

![image](https://user-images.githubusercontent.com/1296369/69330454-e1956900-0c4a-11ea-854f-835602ef0c11.png)

### Additional comments
This works correctly in the core repo
![image](https://user-images.githubusercontent.com/1296369/69330492-f07c1b80-0c4a-11ea-8364-780bbeae88b6.png)
